{
  "abstract" : [
    {
      "text" : "The context supplied with the authenticator continuation request is invalid.",
      "type" : "text"
    }
  ],
  "hierarchy" : {
    "paths" : [
      [
        "doc:\/\/OktaDirectAuth\/documentation\/OktaDirectAuth",
        "doc:\/\/OktaDirectAuth\/documentation\/OktaDirectAuth\/DirectAuthenticationFlowError"
      ]
    ]
  },
  "identifier" : {
    "interfaceLanguage" : "swift",
    "url" : "doc:\/\/OktaDirectAuth\/documentation\/OktaDirectAuth\/DirectAuthenticationFlowError\/invalidContinuationContext"
  },
  "kind" : "symbol",
  "metadata" : {
    "externalID" : "s:14OktaDirectAuth0B23AuthenticationFlowErrorO26invalidContinuationContextyA2CmF",
    "fragments" : [
      {
        "kind" : "keyword",
        "text" : "case"
      },
      {
        "kind" : "text",
        "text" : " "
      },
      {
        "kind" : "identifier",
        "text" : "invalidContinuationContext"
      }
    ],
    "modules" : [
      {
        "name" : "OktaDirectAuth"
      }
    ],
    "role" : "symbol",
    "roleHeading" : "Case",
    "symbolKind" : "case",
    "title" : "DirectAuthenticationFlowError.invalidContinuationContext"
  },
  "primaryContentSections" : [
    {
      "declarations" : [
        {
          "languages" : [
            "swift"
          ],
          "platforms" : [
            "macOS"
          ],
          "tokens" : [
            {
              "kind" : "keyword",
              "text" : "case"
            },
            {
              "kind" : "text",
              "text" : " "
            },
            {
              "kind" : "identifier",
              "text" : "invalidContinuationContext"
            }
          ]
        }
      ],
      "kind" : "declarations"
    }
  ],
  "schemaVersion" : {
    "major" : 0,
    "minor" : 3,
    "patch" : 0
  },
  "sections" : [

  ],
  "variants" : [
    {
      "paths" : [
        "\/documentation\/oktadirectauth\/directauthenticationflowerror\/invalidcontinuationcontext"
      ],
      "traits" : [
        {
          "interfaceLanguage" : "swift"
        }
      ]
    }
  ]
, 
"references": {
"doc://OktaDirectAuth/documentation/OktaDirectAuth": {
  "abstract" : [
    {
      "text" : "Authenticate users using Oktaâ€™s Direct Authentication API, enabling native sign-in experiences supporting Multi-Factor Authentication (MFA).",
      "type" : "text"
    }
  ],
  "identifier" : "doc:\/\/OktaDirectAuth\/documentation\/OktaDirectAuth",
  "kind" : "symbol",
  "role" : "collection",
  "title" : "OktaDirectAuth",
  "type" : "topic",
  "url" : "\/documentation\/oktadirectauth"
},
"doc://OktaDirectAuth/documentation/OktaDirectAuth/DirectAuthenticationFlow": {
  "abstract" : [
    {
      "text" : "An authentication flow that implements the Okta Direct Authentication API.",
      "type" : "text"
    }
  ],
  "fragments" : [
    {
      "kind" : "keyword",
      "text" : "class"
    },
    {
      "kind" : "text",
      "text" : " "
    },
    {
      "kind" : "identifier",
      "text" : "DirectAuthenticationFlow"
    }
  ],
  "identifier" : "doc:\/\/OktaDirectAuth\/documentation\/OktaDirectAuth\/DirectAuthenticationFlow",
  "kind" : "symbol",
  "navigatorTitle" : [
    {
      "kind" : "identifier",
      "text" : "DirectAuthenticationFlow"
    }
  ],
  "role" : "symbol",
  "title" : "DirectAuthenticationFlow",
  "type" : "topic",
  "url" : "\/documentation\/oktadirectauth\/directauthenticationflow"
},
"doc://OktaDirectAuth/documentation/OktaDirectAuth/DirectAuthenticationFlowError": {
  "abstract" : [
    {
      "text" : "Errors that may be generated while authenticating using ",
      "type" : "text"
    },
    {
      "identifier" : "doc:\/\/OktaDirectAuth\/documentation\/OktaDirectAuth\/DirectAuthenticationFlow",
      "isActive" : true,
      "type" : "reference"
    },
    {
      "text" : ".",
      "type" : "text"
    }
  ],
  "fragments" : [
    {
      "kind" : "keyword",
      "text" : "enum"
    },
    {
      "kind" : "text",
      "text" : " "
    },
    {
      "kind" : "identifier",
      "text" : "DirectAuthenticationFlowError"
    }
  ],
  "identifier" : "doc:\/\/OktaDirectAuth\/documentation\/OktaDirectAuth\/DirectAuthenticationFlowError",
  "kind" : "symbol",
  "navigatorTitle" : [
    {
      "kind" : "identifier",
      "text" : "DirectAuthenticationFlowError"
    }
  ],
  "role" : "symbol",
  "title" : "DirectAuthenticationFlowError",
  "type" : "topic",
  "url" : "\/documentation\/oktadirectauth\/directauthenticationflowerror"
},
"doc://OktaDirectAuth/documentation/OktaDirectAuth/DirectAuthenticationFlowError/invalidContinuationContext": {
  "abstract" : [
    {
      "text" : "The context supplied with the authenticator continuation request is invalid.",
      "type" : "text"
    }
  ],
  "fragments" : [
    {
      "kind" : "keyword",
      "text" : "case"
    },
    {
      "kind" : "text",
      "text" : " "
    },
    {
      "kind" : "identifier",
      "text" : "invalidContinuationContext"
    }
  ],
  "identifier" : "doc:\/\/OktaDirectAuth\/documentation\/OktaDirectAuth\/DirectAuthenticationFlowError\/invalidContinuationContext",
  "kind" : "symbol",
  "role" : "symbol",
  "title" : "DirectAuthenticationFlowError.invalidContinuationContext",
  "type" : "topic",
  "url" : "\/documentation\/oktadirectauth\/directauthenticationflowerror\/invalidcontinuationcontext"
}
}
}